docs/management/admin/response-actions.asciidoc

```diff
@@ -92,6 +92,23 @@ Required privilege: *Process Operations*
 
 Example: `suspend-process --pid 123 --comment "Suspend suspicious process"`
 
+=== `get-file`
+
+Retrieve a file from a host. You must include the following parameter to specify the file's location on the host:
+
+* `--path` : The file's full path (including the file name).
+
+Required privilege: *File Operations*
+
+Example: `get-file --path "/full/path/to/file.txt" --comment "Possible malware"`
+
+[NOTE]
+====
+When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a malware prevention alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`), which appears under *Highlighted fields* in the alert details flyout. Then paste the value into the `--path` parameter. 
+
+Quarantined files are downloaded in a password-protected .zip archive to prevent the file from running. Use the passcode `elastic` to open the .zip in a safe environment.
```

Comment:
This is accurate but since all files are downloaded in a password protected zip file (not just quarantined files) - we may want to also indicate that outside of this "help" section. 

Also - it might be helpful to indicate in this getfile section that the osquery manager integration can be used to get insight into files/directories on a host (outside of just the quarantined file workflow).

Reply:
@caitlinbetz Great points, I'll make some tweaks and push another commit soon. Thanks!

Reply:
Added in https://github.com/elastic/security-docs/pull/3043/commits/53b95e09156e3a83f2ee612269ce48c777c01827